packages/ui/Touchable/Hold.js

```diff
@@ -88,12 +94,12 @@ class Hold {
 		this.holdConfig = null;
 	}
 
-	enter = () => {
+	enter = (coords) => {
 		if (!this.isHolding()) return;
 
 		const {resume} = this.holdConfig;
 
-		if (resume) {
+		if (resume && this.isWithinTolerance(coords)) {
```

Comment:
Do you need to check if `cancelOnMove` is set before checking coordinates??

Reply:
It doesn't seem to matter in this case but I think there is still some unexpected behaviors with the config options. I thought this was necessary only because it was noticeable in the current test case whereas `cancelOnMove` isn't used in the TC.

Reply:
Hmm, something seems off to me from that description.  Maybe I'll have to dig into it some.  I thought that the move tolerance only applied when `cancelOnMove` was set.